### PR TITLE
Add task metadata

### PR DESCRIPTION
**Pull request by jluethi**
This PR adds the new task metadata & updates to template version 0.4.2

### CODE REVIEW

**Comment by jluethi** [comment]
We may have to fix numpy<2 for the installation script to work. That did the trick for me locally. Maybe @lorenzocerrone there is a better way to do it? Or are the conda dependencies known to need numpy<2 anyway?

**Comment by jluethi** [comment]
The CI still fails, but that's the same in current main

**Comment by lorenzocerrone** [comment]
> We may have to fix numpy<2 for the installation script to work. That did the trick for me locally. Maybe @lorenzocerrone there is a better way to do it? Or are the conda dependencies known to need numpy<2 anyway?

I don’t think it’s a known thing for Conda. But I think some of the ilastik dependencies (vigra) might need an older numpy. 

pinning it should work! :)

**Comment by lorenzocerrone** [comment]
> The CI still fails, but that's the same in current main (but the actual error log from there has disappeared)

I haven’t managed to make it work, but it should work locally

**Comment by jluethi** [comment]
@lorenzocerrone have a look when you get a chance and merge if you're happy with it :)

File: create_env_script.sh [hunk 1]
```diff
@@ -14,15 +14,15 @@ else
...
+echo "Creating environment $ENVNAME"
 $COMMMAND run $COMMMAND create $LOCATION \
                                --override-channels \
                                -c pytorch \
                                -c ilastik-forge \
                                -c conda-forge $PYTHON ilastik \
                                --no-channel-priority --yes
 
-echo "Installing plantseg-tasks version $VERSION"
+echo "Installing ilastik-tasks version $VERSION"
```
Comment (lorenzocerrone) [inline create_env_script.sh:25]: Thanks for spotting this! :)

**Review by lorenzocerrone — APPROVED** [review]

**Pull request merged**